frontend/src/routes/app-settings.tsx

```diff
@@ -194,6 +208,12 @@ function AppSettingsScreen() {
             step={1}
             className="w-full max-w-[680px]" // Match the width of the language field
           />
+
+          <SystemPromptSelector
+            name="system-prompt-selector"
+            defaultValue={settings.SYSTEM_PROMPT || "system_prompt.j2"}
+            onChange={checkIfSystemPromptHasChanged}
+          />
```

Comment:
@OpenHands the SystemPromptSelector is not properly working. After clicking save, the value was NOT properly propagated